### PR TITLE
[opt](docs) Opt documents for datatypes and compaction

### PR DESCRIPTION
**Pull request by xiaokang**
1. reorg datetypes order
2. opt JSON type docs
3. add single replica and time_series compaction docs

### CODE REVIEW

**Comment by shasevendow** [comment]
1.  All “compaction” words between two Chinese characters must be title-case, i.e. "Compaction".
2. All English titles need to be sentence-case. For example, "## Vertical Compaction" needs to be "Vertical compaction", and "### example" needs to be "### Example".
3. In line 29 for the JSON.md file, "1M" needs to be "1 MB", "2G" needs to be "2 GB", and "be" needs to be "BE".

**Review by shasevendow — APPROVED** [review]
LGTM

File: docs/admin-manual/compaction.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ under the License.
...
 Doris writes data through a structure similar to LSM-Tree, and continuously merges small files into large ordered files through compaction in the background. Compaction handles operations such as deletion and updating. 
 
-Appropriately adjusting the compaction strategy can greatly improve load and query efficiency. Doris provides the following two compaction strategies for tuning:
+Appropriately adjusting the compaction strategy can greatly improve load and query efficiency. Doris provides the following compaction strategies for tuning:
 
 
-## Vertical Compaction
+## Vertical compaction
 
 <version since="1.2.2">
```
Comment (luzhijing) [inline docs/admin-manual/compaction.md:37]: delete version tag.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-types/Data-Types/JSON.md [hunk 2]
```diff
@@ -24,24 +24,61 @@ specific language governing permissions and limitations
...
-<version since="1.2.0">
+默认支持1048576 字节（1 MB），可调大到 2147483643 字节（2 GB），可通过 BE 配置`String_type_length_soft_limit_bytes` 调整。
 
-</version>
+    与普通 String 类型存储的 JSON 字符串相比，JSON 类型有两点优势
+    1. 数据写入时进行 JSON 格式校验
+    2. 二进制存储格式更加高效，通过json_extract等函数可以高效访问JSON内部字段，比get_json_xx函数快几倍
 
-注意：在1.2.x版本中，JSON类型的名字是JSONB，为了尽量跟MySQL兼容，从2.0.0版本开始改名为JSON，老的表仍然可以使用。
+    注意：在1.2.x版本中，JSON 类型的名字是JSONB，为了尽量跟MySQL兼容，从2.0.0版本开始改名为JSON，老的表仍然可以使用。
```
Comment (luzhijing) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-types/Data-Types/JSON.md:35]: ::: Caution
xxx
:::

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0/sql-manual/sql-reference/Data-Types/JSON.md [hunk 3]
```diff
@@ -24,24 +24,61 @@ specific language governing permissions and limitations
...
-<version since="1.2.0">
+默认支持1048576 字节（1 MB），可调大到 2147483643 字节（2 GB），可通过 BE 配置`String_type_length_soft_limit_bytes` 调整。
 
-</version>
+    与普通 String 类型存储的 JSON 字符串相比，JSON 类型有两点优势
+    1. 数据写入时进行 JSON 格式校验
+    2. 二进制存储格式更加高效，通过json_extract等函数可以高效访问JSON内部字段，比get_json_xx函数快几倍
 
-注意：在1.2.x版本中，JSON类型的名字是JSONB，为了尽量跟MySQL兼容，从2.0.0版本开始改名为JSON，老的表仍然可以使用。
+    注意：在1.2.x版本中，JSON 类型的名字是JSONB，为了尽量跟MySQL兼容，从2.0.0版本开始改名为JSON，老的表仍然可以使用。
```
Comment (luzhijing) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0/sql-manual/sql-reference/Data-Types/JSON.md:35]: Same as above.

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**